src/tab.js

```diff
@@ -171,8 +171,14 @@ Object.assign(SideTab, {
     }
     return false;
   },
-  isCloseButtonEvent(e) {
-    return e.target && e.target.classList.contains("tab-close");
+  isCloseButtonEvent(e, checkExplicitOriginal) {
+    if (!e.target) {
+      return;
+    }
+    if (!checkExplicitOriginal) {
+      return e.target.classList.contains("tab-close");
```

Comment:
Okay, it is reasonable.
A question, do you like / mind like the "if (!e.target) return;" codestyle?